### PR TITLE
status: 2023q1: core: clarity, consistency

### PR DESCRIPTION
**Pull request by grahamperrin**
Respond to <https://lists.freebsd.org/archives/dev-commits-doc-all/2023-March/001806.html> from @ceri. 

Whilst here: 

* first and third items use the phrase Core Team, the second item might reasonably do the same

* where the fourth item uses shorter phrases Core, src and source, source can be shortened to src.

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/core.adoc
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/core.adoc:11]: > … delegation's conclusions …

Would what's below be better? 

> … conclusions from the December meeting…

**Comment by grahamperrin** [comment]
@bapt @bsdbcr @emaste @bsdjhb @lwhsu @evadot @tcberner @0mp please, can one of you review this? As a Core Team member. Thanks. 

Contexts: 

1. last month's <https://www.freebsd.org/status/report-2022-10-2022-12/#_freebsd_core_team>
2. today's <https://lists.freebsd.org/archives/dev-commits-doc-all/2023-March/001806.html>.

**Review by lsalvadore — APPROVED** [review]
I approve, but please do not proceed without an approval from core or core secretary.
Moreover, rather than asking for review all members of core, asking the core secretary (@sergio-carlavilla) is probably more appropriate. Indeed, according to https://www.freebsd.org/administration/#t-core-secretary, the secretary is "responsible for documenting the work done by core" and "is also responsible for writing and sending out monthly status reports to the FreeBSD Developer community, containing a summary of core’s latest decisions and actions".

**Comment by grahamperrin** [comment]
Thanks, 

> core secretary

My mistake was visiting <https://www.freebsd.org/administration/#t-core>, which is both memorable, and in Firefox history. 

There's no mention of a Secretary, which I did find surprising. 

(I could have paged up, to reveal the table of contents, however I was more focused on finding GitHub identities of the named members.)

**Comment by sergio-carlavilla** [comment]
Thanks for the change, go ahead!

**Comment by grahamperrin** [comment]
Thanks!

----

@lsalvadore 

> I approve, …

I think, for commit purposes we should have: 

`Approved by: carlavilla, status (salvadore)` 

– for this review. Agreed? 

----

In the two commits that I made around fourteen hours ago, I omitted the `status` (team) context. Sorry.

**Comment by lsalvadore** [comment]
> I think, for commit purposes we should have:
> 
> `Approved by: carlavilla, status (salvadore)`
> 
> – for this review. Agreed?

In general, being more specific is better (but do not overthink it either), thus in this case I would use:

`Approved by: carlavilla (core secretary), status (salvadore)`

> In the two commits that I made around fourteen hours ago, I omitted the `status` (team) context. Sorry.

No problem. If you missed something important or made a mistake in a commit message, you can reply to the commit email with the additional details or the correction, so that the information is accessibile somewhere for reference in the future. You do not need to do it in this case, but if you want you can.

**Comment by grahamperrin** [comment]
<https://github.com/freebsd/freebsd-doc/pull/147#issuecomment-1492888979>

> My mistake … There's no mention of a Secretary, …

For others to not make a similar mistake: 

* https://github.com/freebsd/freebsd-doc/pull/153

**Pull request closed without merging**
